src/test/java/GmailClientTest.java

```diff
@@ -21,7 +21,6 @@
 /** Test GmailClient static functions */
 @RunWith(JUnit4.class)
 public final class GmailClientTest {
-
```

Comment:
Nit: you can leave this as is, but for future reference, try to leave this kind of refactoring (same about `JsonUtilityTest.java`) for a PR that contains no logic. (Especially because both this and `JsonUtilityTest.java` don't contain any change in logic.)

Reply:
Got it - sorry about that!